include/uv-nuttx.h

```diff
@@ -43,7 +43,17 @@
 #include <errno.h>
 
 #include <netdb.h>
-#include <sys/uio.h>
+#if !defined(__TIZENRT__)
```

Comment:
Instead adding Tizen RT releated code to uv-nuttx.h, I would like to make uv-tizenrt.h then collect codes for Tizen RT and put uv-tizenrt.h.